README.md

```diff
@@ -326,6 +329,8 @@ to Miri failing to detect cases of undefined behavior in a program.
   as out-of-bounds accesses) first.  Setting this flag means Miri can miss bugs
   in your program.  However, this can also help to make Miri run faster.  Using
   this flag is **unsound**.
+* `-Zmiri-disable-weak-memory-emulation` disables the emulation of some C++11 weak
+  memory effects.
```

Comment:
Something odd that happens now is that even just passing this flag will disable some UB detection, namely mixed-size accesses, right?
Maybe that is a sign that we should not call this UB, but call it "unsupported" (`throw_unsup_format`) by weak memory emulation?